ibis/backends/duckdb/__init__.py

```diff
@@ -987,7 +987,7 @@ def list_tables(
         return self._filter_with_like(out[col].to_pylist(), like)
 
     def read_postgres(
-        self, uri: str, table_name: str | None = None, schema: str = "public"
+        self, uri: str, *, table_name: str | None = None, schema: str = "public"
```

Comment:
```suggestion
        self, uri: str, *, table_name: str | None = None, database: str = "public"
```

sorry, just caught this.  we want to avoid re-introducing this language